scripts/controllers/searchfilter_controller.ts

```diff
@@ -0,0 +1,49 @@
+
```

Comment:
naming nit:

`SearchFilterController` should be in a file named `search_filter_controller.ts`